mathics/builtin/numbers/algebra.py

```diff
@@ -470,12 +483,9 @@ class FullSimplify(Simplify):
     """
 
     rules = {
-        "FullSimplify[list_List]": "FullSimplify /@ list",
-        "FullSimplify[rule_Rule]": "FullSimplify /@ rule",
-        "FullSimplify[eq_Equal]": "FullSimplify /@ eq",
-        "FullSimplify[list_List, assum_]": "FullSimplify[#1, assum]& /@ list",
-        "FullSimplify[rule_Rule, assum_]": "FullSimplify[#1, assum]& /@ rule",
-        "FullSimplify[eq_Equal, assum_]": "FullSimplify[#1, assum]& /@ eq",
+        "FullSimplify[list_List, OptionsPattern[]]": "FullSimplify /@ list",
+        "FullSimplify[rule_Rule, OptionsPattern[]]": "FullSimplify /@ rule",
```

Comment:
Isn't it the behavior of a Listable function?

Reply:
But is not listable...